src/mock-card.ts

```
@@ -0,0 +1,355 @@
+    clientId: string,
+    clientSecret: string,
+    apiKey: string,
+    host?: string,
+  ) {
+    this.clientId = clientId;
+    this.clientSecret = clientSecret;
+    this.apiKey = apiKey;
+    this.host = host || "https://openapi.investec.com";
+    this.token = "";
+    this.expiresIn = new Date();
+    return this;
```

Biome (1.9.4)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor is ignored.

(lint/correctness/noConstructorReturn)